doc/integration.md

```diff
@@ -103,10 +104,87 @@ Building upon the code sample from the previous section, consider the following
 
 As you can see, your first order of business will usually be to dismiss the registration view upon any of the events occurring. Then, depending on the event, you will want to take further action such as storing the user token or displaying an error message.
 
+You should also declare that your view controller implements the protocol by writing something along the following lines:
+
+```objective-c
+@interface MainViewController : UIViewController<KODRegistrationViewControllerDelegate, ...>
+// Various interface definitions
+@end
+```
+
+As hinted above, the KODRegistrationViewControllerDelegate may not be the only protocol that your view controller adheres to.
```

Comment:
I think it's better if the "UIViewController<KODRegistrationViewControllerDelegate, ...>" is replaced with "UIViewController<KODRegistrationViewControllerDelegate>", and line 115 is omitted, because it is too obvious.